--- /home/nooneknows/projects/nooneknowspeter/cpplings/solutions/00_intro/01_intro.cpp	2026-07-26 14:29:38.087903583 -0400
+++ /home/nooneknows/projects/nooneknowspeter/cpplings/exercises/00_intro/01_intro.cpp	2026-08-02 12:27:15.705841063 -0400
@@ -1,7 +1,3 @@
-// TODO: We sometimes encourage you to keep trying things on a given exercise
-// even after you already figured it out. If you got everything working and feel
-// ready for the next exercise, enter `n` in the terminal.
-
 #include <iostream>
 
 void splashScreen()
