--- /home/nooneknows/projects/nooneknowspeter/cpplings/solutions/02_functions/03_functions.cpp	2026-08-15 23:26:16.924497613 -0400
+++ /home/nooneknows/projects/nooneknowspeter/cpplings/exercises/02_functions/03_functions.cpp	2026-08-15 23:26:13.651461795 -0400
@@ -24,7 +24,7 @@
 
 // TODO: Add the function datatype and name.
 // Add two parameters, one that takes in a boolean and the other a float.
-void voidFunction(bool isTruthy, float randomValue) {
+voidFunction(bool isTruthy, float randomValue) {
   try {
     if (typeid(isTruthy).name() != typeid(bool).name()) {
       throw std::runtime_error("The first parameter is not a boolean.\n");
