--- /home/nooneknows/projects/nooneknowspeter/cpplings/solutions/02_functions/04_functions.cpp	2026-08-15 23:25:58.763298883 -0400
+++ /home/nooneknows/projects/nooneknowspeter/cpplings/exercises/02_functions/04_functions.cpp	2026-08-15 23:25:54.874256331 -0400
@@ -50,6 +50,6 @@
 
 int main(int argc, char* argv[]) {
   // TODO: Print out the argc and argv.
-  std::cout << "Number of arguments: " << argc << "\n"
-            << "Arguments: " << *argv << std::endl;
+  std::_ << "Number of arguments: " << argc << "\n"
+         << "Arguments: " << *argv << std::endl;
 }
