--- /home/nooneknows/projects/nooneknowspeter/cpplings/solutions/02_functions/04_functions.cpp	2026-07-26 14:29:38.088765124 -0400
+++ /home/nooneknows/projects/nooneknowspeter/cpplings/exercises/02_functions/04_functions.cpp	2026-08-03 10:38:58.251163384 -0400
@@ -45,5 +45,5 @@
 int main(int argc, char *argv[])
 {
     // TODO: Print out the argc and argv.
-    std::cout << "Number of arguments: " << argc << "\n" << "Arguments: " << *argv << std::endl;
+    std::_ << "Number of arguments: " << argc << "\n" << "Arguments: " << *argv << std::endl;
 }
