--- /home/nooneknows/projects/nooneknowspeter/cpplings/solutions/15_threads/01_threads.cpp	2026-08-02 13:49:54.793094080 -0400
+++ /home/nooneknows/projects/nooneknowspeter/cpplings/exercises/15_threads/01_threads.cpp	2026-08-03 10:41:10.747390754 -0400
@@ -29,7 +29,7 @@
 // Use a mutex to protect the shared resource.
 void incrementSafely()
 {
-    std::lock_guard<std::mutex> lock(mtx);
+    std::_ lock(mtx);
     counter++;
 }
 
