--- patches/solutions/01_variables/02_variables.cpp	2026-03-17 15:26:59.196802975 -0400
+++ exercises/01_variables/02_variables.cpp	2026-03-13 23:19:24.725666031 -0400
@@ -1,21 +1,9 @@
-// A variable's type must be declared before
-// the variable's name.
-//
-// int foo{5};
-// float bar{10.5f}; / float bar{10.5};
-// double baz{20.56};
-
 #include <iostream>
 
 void variables02()
 {
-    // TODO: Add a missing type and value.
+    // TODO: Add the missing keyword
     int x{5};
 
-    std::cout << "x=" << x;
-}
-
-int main()
-{
-    variables02();
+    std::cout << x;
 }
